SWLOR.Game.Server/Service/CombatPoint.cs

```diff
@@ -24,33 +24,6 @@ public static class CombatPoint
         /// </summary>
         private static readonly Dictionary<uint, HashSet<uint>> _playerToCreatureTracker = new Dictionary<uint, HashSet<uint>>();
 
-        /// <summary>
```

Comment:
This is a good call. There is a method in StandardItemConfiguration.cs which adds the item property in the first place. Its name is ApplyOnHitProperty. Does it make sense to get rid of it as well or do we still have a use for it?

Reply:
Suggest we leave it for now but kill it before launch if we're sure we don't need to put it back :)